packages/common/types/PortalTypes.ts

```diff
@@ -335,3 +336,20 @@ export interface ClasslistTransport {
     SEC: string;
     LAB: string;
 }
```

Comment:
Not too sure how this is being used yet, but might be useful.

Reply:
I think it will be good practice to just maintain the `custom` fields on all of these objects; they give a lightweight way for a course to to extend an object without breaking any type checking for the core fields.